tests/InputFiles/GrMhd/GhValenciaDivClean/GhMhdTovStar.yaml

```diff
@@ -101,21 +101,21 @@ EvolutionSystem:
         AnalyticPrescription: *InitialData
     DampingFunctionGamma0:
       GaussianPlusConstant:
-        Constant: 1.0
-        Amplitude: 0.0
-        Width: 1.0
+        Constant: 0.01
+        Amplitude: 0.12
+        Width: 7.884855
         Center: [0.0, 0.0, 0.0]
     DampingFunctionGamma1:
       GaussianPlusConstant:
-        Constant: -1.0
-        Amplitude: 0.0
-        Width: 1.0
+        Constant: -0.999
+        Amplitude: 0.999
+        Width: 30.0
         Center: [0.0, 0.0, 0.0]
     DampingFunctionGamma2:
       GaussianPlusConstant:
-        Constant: 1.0
-        Amplitude: 0.0
-        Width: 1.0
+        Constant: 0.01
+        Amplitude: 1.2
+        Width: 7.884855
```

Comment:
Would it be worth adding a comment like `# This number comes from the radius of the star in code units.`?